keycloak/keycloak-admin-ui
# Realm settings(Email): Adds email tab to realm settings section

## Motivation

https://issues.redhat.com/browse/APPDUX-935

## Verification Steps


1. Go to Realm settings -> Email tab
2. Populate fields and test checkboxes/switches
3. Click save and navigate away from the page.
4. Return to the Email tab and verify the data is stored/saved correctly.

## Checklist:
